modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lcao-using-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} on both the seed and target cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the {rh-rhacm} Operator 2.9.2 or later.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `lcao-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-lifecycle-agent-subscription
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  name: lifecycle-agent
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f lcao-subscription.yaml
+----
+
+.Verification
+
+. Verify that the installation succeeded by inspecting the CSV resource:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-operators
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+NAME                                           DISPLAY                    VERSION               REPLACES                           PHASE
+lifecycle-agent-operator.v{product-version}.0   Lifecycle Agent Operator   {product-version}.0                Succeeded
```

Comment:
same, see suggestion in https://github.com/openshift/openshift-docs/pull/69809#discussion_r1504699348

for me, it now renders like the below image, it should be as in the suggestion ^^
![image](https://github.com/openshift/openshift-docs/assets/13664224/11a46e4a-5a2a-4430-a506-ecac7bea5b69)

Reply:
DISPLAY:
s/Lifecycle Agent Operator/Openshift Lifecycle Agent/g

NAME:
s/lifecycle-agent-operator.v{product-version}.0/lifecycle-agent.v{product-version}.0/g